compile/fanduel_single_game_template.py

```diff
@@ -28,50 +26,65 @@
 # stack_info = stack_auto[['s_z', 'p_z', 'z', 'points', 'salary', 'stacks', 'sp_mu']].sort_values(by='points', ascending=False)
 
 
-# def build_lineups(self, 
-#                       lus = 150, 
-#                       index_track = 0, 
+# def build_lineups(self,
+#                       lus = 150,
```

Comment:
Making a comment on a random line so you can see how this works. Nothing wrong here, but you can see how the context helps around “oh found an issue here”

Reply:
You can comment here, and when a change has been agreed to, you can make it and then resolve the conversation when you’re done. A nice todo

Reply:
Thank you!